api/api.go

```diff
@@ -810,3 +830,42 @@ query namespaceOrbs ($namespace: String, $after: String!) {
 
 	return orbs, nil
 }
+
+// IntrospectionQuery makes a query on the API asking for bits of the schema
+// This query isn't intended to get the entire schema, there are better tools for that.
+func IntrospectionQuery(ctx context.Context, logger *logger.Logger) (*IntrospectionResponse, error) {
+	var response struct {
+		IntrospectionResponse
+	}
+
+	query := `query IntrospectionQuery {
```

Comment:
What do you think of moving queries out into their own `.graphql` files? It would make our implementation code more focused on what Go does with the response, and we could more directly reference these queries with `cat ... | circleci query -` if we want to interact with them in the terminal.

Reply:
We have talked about doing exactly that. I love the idea also because it would help this be the reference implementation of the API and be a great learning tool for our users.